ehrapy/preprocessing/_encode.py

```diff
@@ -72,11 +72,21 @@ def encode(
         >>> # encode col1 and col2 using label encoding and encode col3 using one hot encoding
         >>> adata_encoded = ep.pp.encode(adata,
         >>>                              autodetect=False,
-        >>>                              encodings={'label_encoding': ['col1', 'col2'], 'one_hot_encoding': ['col3']})
+        >>>                              encodings={'label': ['col1', 'col2'], 'one-hot': ['col3']})
     """
     if isinstance(data, AnnData):
         # basic type checking for passed parameters when encoding a single AnnData object
-        _check_anndata_input_type(autodetect, encodings)
+        if not isinstance(autodetect, bool):
```

Comment:
Type hint for autodetect is bool or dict; what is this check for?

Reply:
Removed.